src/main/java/org/web3j/console/project/java/UnitTestCreator.java

```diff
@@ -59,12 +60,12 @@ public static void main(String[] args) {
         CommandLine.run(new UnitTestCLIRunner(), args);
     }
 
-    void generate() throws IOException {
+    public void generate() throws IOException {
         List<Class> compiledClasses = new ClassProvider(new File(wrapperPath)).getClasses();
         compiledClasses.forEach(
                 compiledClass -> {
                     try {
-                        new UnitClassGenerator(
+                        new JavaClassGenerator(
```

Comment:
Is this change right? Seems odd to go from unit to java?

Reply:
fixed